src/app/analysis-types-list/analysis-types-list/analysis-types-list.component.html

```diff
@@ -0,0 +1,26 @@
+<mat-accordion [displayMode]="flat">
+<mat-expansion-panel *ngFor="let type of types, let i = index"
+                     (opened)="panelOpenState[i] = true"
+                     (closed)="panelOpenState[i] = false">
+  <mat-expansion-panel-header>
+  <mat-panel-title>
+    <i
+      class="fa fa-cogs"
+      aria-hidden="true">
+    </i>
+    <div class="font-cont">
+      {{type}}
+    </div>
+  </mat-panel-title>
+  <mat-panel-description>
+    {{panelOpenState[i] ? 'less...' : 'more...'}}
```

Comment:
These dots with current width of the sidebar are pretty wide. We can go for simple *less* and *more* without dots.